packages/bruno-app/src/components/MarkDownEditor/index.jsx

```diff
@@ -0,0 +1,155 @@
+import MarkdownIt from 'markdown-it';
+import * as MarkdownItReplaceLink from 'markdown-it-replace-link';
```

Comment:
addressed merged conflicts to work with  https://github.com/usebruno/bruno/pull/2375